app/src/main/java/com/realityexpander/tasky/agenda_feature/presentation/components/EditTextModal.kt

```diff
@@ -0,0 +1,182 @@
+package com.realityexpander.tasky.agenda_feature.presentation.components
+
+import android.content.res.Configuration
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.interaction.MutableInteractionSource
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.text.KeyboardOptions
+import androidx.compose.material.*
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.outlined.ChevronLeft
+import androidx.compose.runtime.*
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.focus.FocusRequester
+import androidx.compose.ui.focus.focusRequester
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.input.KeyboardType
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.tooling.preview.datasource.LoremIpsum
+import androidx.compose.ui.unit.dp
+import com.realityexpander.tasky.core.presentation.common.modifiers.DP
+import com.realityexpander.tasky.core.presentation.theme.TaskyGreen
+import com.realityexpander.tasky.core.presentation.theme.TaskyTheme
+
+@OptIn(ExperimentalMaterialApi::class)
+@Composable
+fun EditTextModal(
+    title: String,
+    text: String = "",
```

Comment:
A bit nitpicky, but I'd not use default values for parameters that make up the core functionality of a composable, since that increases the likelihood that you forget to assign them and then first need to check why

Reply:
OK good point. Will address.